libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -257,6 +264,72 @@ instance HasDeprecatedFeatureName 'TeamFeatureDigitalSignatures where
 typeTeamFeatureName :: Doc.DataType
 typeTeamFeatureName = Doc.string . Doc.enum $ cs . toByteString' <$> [(minBound :: TeamFeatureName) ..]
 
+----------------------------------------------------------------------
+-- TeamFeatureTTLValue
+
+-- Using Word to avoid dealing with negative numbers.
+-- Ideally we would also not support zero.
+-- Currently a TTL=0 is ignored on the cassandra side.
+data TeamFeatureTTLValue
+  = TeamFeatureTTLSeconds Word
+  | TeamFeatureTTLUnlimited
+  deriving stock (Eq, Show, Generic)
+
+deriving instance ToJSON TeamFeatureTTLValue
```

Comment:
do you even need these?  (just wondering, i don't mind them being there just in case.)

Reply:
`ToBytestring` is used to serialise it for Bilge. `ToHttpApiData` on the other hand, doesn't seem necessary. 👍